src/libcore/mem.rs

```diff
@@ -1056,12 +1056,22 @@ impl<T: ?Sized> DerefMut for ManuallyDrop<T> {
 /// This is exploited by the compiler for various optimizations, such as eliding
```

Comment:
Says "equivalent code with `MaybeUninit`" at the top, can you add `<T>`?
(or more generally, do a search for MaybeUninit inside `` and replace with `MaybeUninit<T>`?)